jupyter_cache/executors/basic.py

```diff
@@ -57,12 +57,13 @@ class JupyterExecutorBasic(JupyterExecutorAbstract):
 
     The execution is split into two methods: `run` and `execute`.
     In this way access to the cache can be synchronous, but the execution can be
-    multi/async processed.
+    multi/async processed. Takes timeout parameter in seconds for execution
```

Comment:
Could you add a short section on "timeout" under the execution documentation here: https://jupyter-cache.readthedocs.io/en/latest/using/api.html#execution

And mention that you can specify a `timeout` either via a the call to jupyter-cache, or via the `execution: timeout:` metadata field?

Reply:
I had added a small Note there https://github.com/executablebooks/jupyter-cache/pull/48/files#diff-7d5c2179cfd8d418561a37c077b98278R1063 . But, creating a small section sounds better

Reply:
![image](https://user-images.githubusercontent.com/6542997/83091215-08db6000-a0de-11ea-8357-aab636024c9a.png)

@choldgraf what do you think?